lib/prediction_class.py

```diff
@@ -432,7 +431,7 @@ def valid_allele_names(self):
 
     def iedb_executable_params(self, iedb_executable_path, method, allele, input_file, epitope_length):
         allele = allele.replace('-DPB', '/DPB').replace('-DQB', '/DQB')
-        return "{} {} {} {}".format(iedb_executable_path, method, allele, input_file)
+        return "{} {} {} {} {}".format(iedb_executable_path, method, allele, input_file, epitope_length)
```

Comment:
Over on the class I side, this is:
https://github.com/griffithlab/pVACtools/blob/9867b5ee04c9aa853ea140485e14727ea84387f1/lib/prediction_class.py#L355-L356

Should this use the `str(epitope_length)`, too?  (Kind of surprising to me that the ordering is different, as well.)

Reply:
the `str` part is actually not needed (`format` will cast it for you). I believe this is a leftover from before `format` was a thing in python and I never removed it when I switched this to using `format`. 

The epitope_length addition on Class II is a new thing that was added to IEDB only recently so they appended that argument. It used to just default to 15. They probably didn't want to break backward compatibility. For Class II it also isn't required (it defaults to the old "standard" 15). Meanwhile it was always available for Class I so it got a more preferable position and there it is required.

Reply:
Thanks for the explanation!  I figured `str` couldn't be that important here or the tests would fail horribly 😄